docs/sdks/uid2-sdk-ref-ios.md

```diff
@@ -93,19 +93,27 @@ Add the following dependency to Package.swift:
 
 ``` javascript
 dependencies: [
-.package(url: \"https://github.com/IABTechLab/uid2-ios-sdk.git\", from: "0.3.0"),
+.package(url: \"https://github.com/IABTechLab/uid2-ios-sdk.git\", from: "1.0.0"),
 ]
 ```
 
 ### Installing with Xcode
 
-Add the following dependency via Xcode:
+Add the following dependency via Xcode Package Dependencies:
```

Comment:
@dcaunt i want to replace the screenshot as shown on https://unifiedid.com/docs/sdks/uid2-sdk-ref-ios#installing-with-xcode as everytime we make a new release we need to create a new screenshot. 

Please let me know if this description and table below makes the same info clear or not, and if not, how we could improve it. Thanks

Reply:
thanks @dcaunt i have corrected it per your suggestion and i reworded the top description from:

_Add the following dependency via Xcode Package Dependencies:_
to 
_Add the following entry in your application's Package Dependencies:_

which may be clearer